hardware/PrusaResearch/avr/cores/prusa_einsy_rambo/CDC.cpp

```diff
@@ -34,14 +34,14 @@ typedef struct
 static volatile LineInfo _usbLineInfo = { 57600, 0x00, 0x00, 0x00, 0x00 };
 static volatile int32_t breakValue = -1;
 
-bool _updatedLUFAbootloader = false;
+static u8 wdtcsr_save;
 
 #define WEAK __attribute__ ((weak))
 
 extern const CDCDescriptor _cdcInterface PROGMEM;
 const CDCDescriptor _cdcInterface =
 {
-	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,1),
+	D_IAD(0,2,CDC_COMMUNICATION_INTERFACE_CLASS,CDC_ABSTRACT_CONTROL_MODEL,0),
```

Comment:
Why changing `1->0`? Does it influence the USB-CDC communication in some important way?

Reply:
Again it is done by Arduino IDE 1.8.19

Reply:
resolved in https://github.com/prusa3d/Arduino_Boards/pull/16